digital_land/commands.py

```diff
@@ -715,9 +737,91 @@ def add_data(
 
     if user_response != "yes":
         print("Operation cancelled by user.")
-        clear_log(collection_dir, endpoint_resource_info["endpoint"])
         return
 
+    if not cache_dir:
+        cache_dir = Path("var/cache/add_data/")
+    output_path = os.path.join(
+        cache_dir, "transformed", endpoint_resource_info["resource"] + ".csv"
+    )
+    issue_dir = os.path.join(cache_dir, "issue/")
+    column_field_dir = os.path.join(cache_dir, "column_field/")
+    dataset_resource_dir = os.path.join(cache_dir, "dataset_resource/")
+    converted_resource_dir = os.path.join(cache_dir, "converted_resource/")
+    converted_dir = os.path.join(cache_dir, "converted/")
+    output_log_dir = os.path.join(cache_dir, "log/")
+    operational_issue_dir = os.path.join(cache_dir, "performance", "operational_issue")
+
+    os.makedirs(os.path.dirname(output_path), exist_ok=True)
+    os.makedirs(issue_dir, exist_ok=True)
+    os.makedirs(column_field_dir, exist_ok=True)
+    os.makedirs(dataset_resource_dir, exist_ok=True)
+    os.makedirs(converted_resource_dir, exist_ok=True)
+    os.makedirs(converted_dir, exist_ok=True)
+    os.makedirs(output_log_dir, exist_ok=True)
+    os.makedirs(operational_issue_dir, exist_ok=True)
+
+    collection.load_log_items()
+    for pipeline in endpoint_resource_info["pipelines"]:
+        print("======================================================================")
+        print("Run pipeline")
+        print("======================================================================")
+        try:
```

Comment:
We could look at refactoring some stuff to make this easier but might be one for after this work

Reply:
Also this means that all the logs aren't accessible so could help.